grasa_event_locator/urls.py

```diff
@@ -8,7 +8,6 @@
 urlpatterns = [
     path('aboutContact.php', views.aboutContact),
     path('admin.php', views.admin, name='admin_page'),
-    path('admin_activate', views.admin_activate),
```

Comment:
Is this the URL to create an initial admin user?

Reply:
Exactly as Harrison said, it was no longer needed so I removed it